lldb/source/Commands/CommandObjectStats.cpp

```diff
@@ -83,13 +86,15 @@ class CommandObjectStatsDump : public CommandObjectParsed {
 
     void OptionParsingStarting(ExecutionContext *execution_context) override {
       m_all_targets = false;
+      m_summary_only = false;
     }
 
     llvm::ArrayRef<OptionDefinition> GetDefinitions() override {
       return llvm::ArrayRef(g_statistics_dump_options);
     }
 
     bool m_all_targets = false;
```

Comment:
This should get moved into `lldb_private::StatisticsOptions` as an option. Add accessor to `SBStatisticsOptions` so we can get/set this as well.